tests/keras/backend/backend_test.py

```diff
@@ -9,31 +9,32 @@
 from keras.utils.conv_utils import convert_kernel
 import reference_operations as KNP
 
-
 BACKENDS = []  # Holds a list of all available back-ends
 
 try:
     from keras.backend import cntk_backend as KC
+
```

Comment:
It seems there are empty lines added and removed. Can you delete those differences from master?